### PR TITLE
add pgadmin4 to docker-compose

### PR DESCRIPTION
**Pull request by ductm54**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -23,4 +23,12 @@ services:
     volumes:
       - './data/redis:/var/lib/redis/'
     ports:
-      - '127.0.0.1:6379:6379' 
\ No newline at end of file
+      - '127.0.0.1:6379:6379'
+  pgadmin4:
+    image: dpage/pgadmin4
+    ports:
+        - '127.0.0.1:60978:60978'
```
Comment (favadi) [inline docker-compose.yml]: From https://hub.docker.com/r/dpage/pgadmin4, I think the default listening port is 80.

File: docker-compose.yml [hunk 2]
```diff
@@ -23,4 +23,12 @@ services:
...
+    environment:
+        PGADMIN_LISTEN_PORT: 60978
+        PGADMIN_DEFAULT_EMAIL: postgres@localhost
+        PGADMIN_DEFAULT_PASSWORD: secret
```
Comment (favadi) [inline docker-compose.yml]: could you make email, password: reserve_stats@localhost/reserve_stats just like default password?

Comment (ductm54) [reply docker-compose.yml]: changed

File: docker-compose.yml (hunk 1 above)
Comment (ductm54) [reply docker-compose.yml]: updated

**Pull request merged**